### ISSUE TITLE
Feature request: DecryptMedia method for Easy API

### ISSUE THREAD

**Issue by silvaartur** [open-wa/wa-automate-nodejs#751]
A simple method to call /DecryptMedia pass params and decrypt media with base64 return.

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [2.0.7](https://github.com/open-wa/wa-automate-nodejs/compare/2.0.6...2.0.7) (2020-08-28)

- Release 2.0.7 [`4da7842`](https://github.com/open-wa/wa-automate-nodejs/commit/4da7842a3fca63a77225a07ac51cda8c51cc5a26)
- Feature: Implemented `decryptMedia` in `Client` #751 [`a89077c`](https://github.com/open-wa/wa-automate-nodejs/commit/a89077ccb1005331336d791decc88bc99afd11a0)
- Feture: allow custom pop up port for cli [`dbbabe0`](https://github.com/open-wa/wa-automate-nodejs/commit/dbbabe009cbe59a4e5865b052e8d59a0655d46e3)
- Host Notifcation Translation added: `id-id` #743 [`8b731ff`](https://github.com/open-wa/wa-automate-nodejs/commit/8b731ffa3eb93eb2d031bada3f960a9b1811de4f)

**Comment by smashah** [comment]
@silvaartur You can now call the endpoint `decryptMedia` with the following body:

```javascript
{
  "args": {
      "message" : "false_444777777777@c.us_FFCA779E5BCFE525E0612NIJ4NIF34I8"
  }
}
```

The response will return a DataURL.
